### ISSUE TITLE
Make ipxe behavior customizable

### ISSUE THREAD

**Issue by bzub** [talos-systems/sidero#287]
One knob I need is to customize the ipxe response when a server is allocated/installed. Sidero currently returns an ipxe script that executes `exit` in the hopes of booting from disk, but our servers halt booting if this happens, requiring manual intervention. I would change this to returning an HTTP error code if I had such an option, which our servers then try other boot options or reboot as needed.

In short, I'd like to be able to change this:
```
❯ curl -s 'http://assets.management:8081/ipxe?uuid=UUID-OF-INSTALLED-SERVER'
#!ipxe
exit
```

to this:
```
❯ curl -v 'http://assets.management:8081/ipxe?uuid=UUID-OF-INSTALLED-SERVER'
[...]
< HTTP/1.1 404 Not Found
[...]
```

Or something else? `exit` seems to cause issues.

Related: https://ipxe.org/appnote/work_around_bios_halting_on_ipxe_exit
